autotest/common_regression.py

```diff
@@ -440,10 +444,9 @@ def get_mf6_comparison(src):
         comparison type
 
     """
-    action = None
     # Possible comparison - the order matters
-    optcomp = (
-        "compare",
+    OPTCOMP = (
```

Comment:
I like the change to uppercase for defined data.

Reply:
I meant to move this to a top-level variable so other modules could import it, will do so if you agree